js/index.js

```diff
@@ -215,107 +216,161 @@ webix.ready(function(){
           }else{return false}
         }
       }
-      /*onValidationError:function(key, data){
-        webix.message.position = "bottom";
-        webix.message({text:key.toUpperCase()+"  field is incorrect", type:"error"});
-      }*/
     };
+
+  //For list editing ability
+  webix.protoUI({
+    name:"editlist"
+  }, webix.EditAbility, webix.ui.list);
+
   var list = {
-    type:"clean",
-    rows:[
+    view:"editlist",
+    id:"users_list",
+    scheme:{
+      $init:function(obj){
+        if(obj.age < 26){
+          obj.$css = "young_users";
+        }
+      }
+    },
+    editable:true,
+    editor:"text",
+    editValue:"name",
+    template:"#name#, #age#, <span> from </span>#country# <span class='remove_list_item_btn webix_icon mdi mdi-close'></span>",
+    height:300,
+    scrollY:true,
+    scrollX:false,
+    select:true,
+    rules:{
+      name:webix.rules.isNotEmpty
+    },
+    onClick:{
+      remove_list_item_btn:function(e, id){
+        this.remove(id);
```

Comment:
it would be nice to have a confirm message for remove operations

Reply:
Confirmation was added